index.html

```diff
@@ -229,8 +230,9 @@ <h4 class="popup__type">Квартира</h4>
       <button type="button" class="error__button">Попробовать снова</button>
     </div>
   </template>
-  <script type="module" src="js/script.js"></script>
 
+  <script src="/leaflet/leaflet.js"></script>
```

Comment:
```suggestion
  <script src="leaflet/leaflet.js"></script>
```

слеш убери, тут относительный путь